site2/docs/getting-started-standalone.md

```diff
@@ -171,6 +171,8 @@ If you have started Pulsar successfully, you will see `INFO`-level log messages
 
 > #### Tip
 > 
+> * Before starting the Pulsar standalone, you may need to config the advertisedAddress to avoid startup errors. Actually, you can modify the default value of advertisedAddress in "conf/standalone.conf" or use command "bin/pulsar standalone -a 127.0.0.1" to assign the advertisedAddress.
+>
```

Comment:
This should only be added to the 2.4.2 version of the docs. In 2.5 this step won't be necessary anymore.

Reply:
Appreciation, I have added version description in this tip. PTAL

Reply:
@fangpengcheng95 FYI. Here is the link of [2.4.2 verison](https://github.com/apache/pulsar/blob/master/site2/website/versioned_docs/version-2.4.2/getting-started-standalone.md)

Reply:
@jiazhai Thanks a lot for reviews and advice. I have moved the tip to [2.4.2 version](https://github.com/apache/pulsar/blob/master/site2/website/versioned_docs/version-2.4.2/getting-started-standalone.md). PTAL.